app/content/views/upload.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+
+from sentry_sdk import capture_exception
+
+from app.common.azure_blob_storage_handler import uploadBlob
+from app.common.permissions import IsMember
+
+
+@api_view(["POST"])
+@permission_classes([IsMember])
+def upload(request):
+    """ Method for accepting company interest forms from the company page """
+    try:
+        if len(request.FILES) > 1:
+            return Response(
+                {"detail": "Du må kan ikke sende med flere filer"},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        if len(request.FILES) < 1:
+            return Response(
+                {"detail": "Du må sende med en fil i FILE"},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        key = list(request.FILES.keys())[0]
+        blob = request.FILES[key]
+        url = uploadBlob(blob)
```

Comment:
Vi burde sende med container-navnet med i bodyen i requesten slik at man kan bestemme hvilken container filen skal ligge i når man laster den opp

Reply:
Og dermed sende den med i uploadBlob-metoden

Reply:
Litt usikker på dette. Hvor viktig er det å legge blobs i forskjellige containere? Fullt mulig men ser ikke helt behovet hvis det ikke er viktig med forskjellige containere siden det da kompliserer koden unødvendig.

Reply:
Ja forsåvidt. Det spørs helt hvordan vi ønsker å løse det. Jeg har sett for meg egne containere for bilder, dokumenter osv... Men dette kan forsåvidt gjøres uten å sende med i bodyen da du har filtypen ...

Reply:
Ja, for hvis det gir bedre performance å dele inn i forskjellige containere så bør vi gjøre det. Godt poeng med filtype, veldig med på å bruke den løsningen.

Reply:
Går det an å slette filer basert på urlen eller hvordan gjøres det?

Reply:
Hvis man har url til en fil så kan man lese av navn på container og blob som man må ha for å kunne slette filer, men går ikke an uten credentials uansett